cypress/integration-qe/e2e-enterClusterSpecs.spec.js

```diff
@@ -115,3 +125,13 @@ describe('Donwload kubeconfig', () => {
     cy.exec('mv -f ~/Downloads/kubeconfig ~');
   });
 });
+
+describe('Refresh cookie on disk', () => {
+  it('refresh cookie', () => {
```

Comment:
I can see similar case in `e2e-createNewCluster.spec.js`. I guess, motivation behind both of them is the same. Can we unify?

And is this really a test case/scenario? If not, what about using hooks like `beforeEach()` or similar?